### ISSUE TITLE
New install, and nothing is downloading, tasks show immediately.

### ISSUE THREAD

**Issue by ForceConstant** [meeb/tubesync#544]
So I have a new install, and setup tubesync using docker. Tubesync seems to have synced a channel, and put things to download, but no media has actually downloaded. In the tasks section it says 67 tasks to download immediately, but no action. I have restarted the container, but nothing happens. Ideas?

![image](https://github.com/user-attachments/assets/859f99fc-58a7-47df-ad53-00f8e9b726d2)
![image](https://github.com/user-attachments/assets/ce61a86b-01d6-48d8-8440-e0fdd58cb3e4)

**Comment by ForceConstant** [comment]
1. No the tasks are not going down. 
2. That is the whole container logs, doesn't look like anything is happening.
3. I tried doing the reset-tasks, and it worked for a while, and seems to have downloaded all of the metadata, but has left the scheduled tasks to actually download the items.

**Comment by meeb** [comment]
try resetting tasks on the command line, like:

```bash
docker exec -ti tubesync python3 /app/manage.py reset-tasks
```

**Comment by ForceConstant** [comment]
Yes I have previously ran this command to reset-tasks. 

> try resetting tasks on the command line, like:
> 
> ```shell
> docker exec -ti tubesync python3 /app/manage.py reset-tasks
> ```

Yes I have previously ran this to reset tasks. Any way to add debug to see what the tasks are waiting for? Note I also saw in another issue similar issue that was fixed by making sure the timezone was correct inside the container, and have verified this.

**Comment by meeb** [comment]
Yeah obviously something isn't running correctly, the issue is there's nothing in your logs to indicate what the issue is. What processes are running in the container?

**Comment by ForceConstant** [comment]
![image](https://github.com/user-attachments/assets/556870df-e416-4e7c-b4ed-d3b4a1ca7792)

**Comment by meeb** [comment]
That shows that the task runner is running properly. I've no idea really why it's not processing tasks if that is indeed the issue.

**Comment by alvarokrn** [comment]
It's happening the same to me.

**Comment by YourWishes** [comment]
> try resetting tasks on the command line, like:
> 
> ```shell
> docker exec -ti tubesync python3 /app/manage.py reset-tasks
> ```

I'm experiencing the same issue, clean install running through docker compose, tasks are adding fine but none are processing, running this command gives;
![image](https://github.com/user-attachments/assets/fd38d042-3f93-47d1-932d-d82ce897beeb)

**Comment by meeb** [comment]
@YourWishes that error means your database schema is missing or corrupted. This isn't the same issue. What does this output?

```bash
$ docker exec -ti tubesync python3 /app/manage.py migrate
```

**Comment by tcely** [comment]
I believe this was fixed by #690.

**Comment by meeb** [comment]
This looks resolved, thanks @tcely !
